protected/views/ethnicGroup/create.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/* @var $this EthnicGroupController */
+/* @var $model EthnicGroup */
+$this->pageTitle = 'Create Ethnic';
+?>
+<h1 class="badge">Create Ethnic</h1>
+<div class="box content admin-content">
+  <div class="large-10 column content admin large-centered">
+    <div class="box admin">
+      <h1 class="text-center">Create Disorder</h1>
```

Comment:
Should this be a h2 instead of a h1?

Reply:
In disorder views, it's h1, should I change those as well?